wire-ios-data-model/Source/Model/User/UserType.swift

```diff
@@ -57,6 +57,9 @@ public protocol UserType: NSObjectProtocol, UserConnections {
     /// The name of the team the user belongs to.
     var teamName: String? { get }
 
+    /// The id of the team the user belongs to.
+    var team: Team? { get }
```

Comment:
I don't think, `team` is required as separate property, since `membership` is required and it contains the team.
I will revert these changes.